README.md

```diff
@@ -8,93 +8,95 @@
+  * [Roman Bapst](https://github.com/RomanBapst)
+* [Fixed Wing Flight Control](https://github.com/PX4/Firmware/labels/fixedwing)
+  * [Roman Bapst](https://github.com/RomanBapst)
+* OS / NuttX [David Sidrane](https://github.com/davids5)
+* Driver Architecture [Daniel Agar](https://github.com/dagar)
+* Commander Architecture [Julian Oes](https://github.com/julianoes)
+* [UAVCAN](https://github.com/PX4/Firmware/labels/uavcan) [Daniel Agar](https://github.com/dagar)
+* [State Estimation](https://github.com/PX4/Firmware/issues?q=is%3Aopen+is%3Aissue+label%3A"state+estimation") - [Paul Riseborough](https://github.com/priseborough)
+* Vision based navigation
+  * [Julian Kent](https://github.com/jkflying)
+* Obstacle Avoidance - [Martina Rivizzigno](https://github.com/mrivi)
+* RTPS/ROS2 Interface - [Nuno Marques](https://github.com/TSC21)
+
+See also [About Us](http://px4.io/about-us/#development_team) \(px4.io\) and the [contributors list](https://github.com/PX4/Firmware/graphs/contributors) \(Github\).
 
 ## Supported Hardware
 
 This repository contains code supporting these boards:
-  * [Snapdragon Flight](https://docs.px4.io/en/flight_controller/snapdragon_flight.html)
-  * [Intel Aero](https://docs.px4.io/en/flight_controller/intel_aero.html)
-  * [Raspberry PI with Navio 2](https://docs.px4.io/en/flight_controller/raspberry_pi_navio2.html)
-  * [Parrot Bebop 2](https://dev.px4.io/en/advanced/parrot_bebop.html)
-  * FMUv2
-    * [Pixhawk](https://docs.px4.io/en/flight_controller/pixhawk.html)
-    * [Pixhawk Mini](https://docs.px4.io/en/flight_controller/pixhawk_mini.html)
-    * [Pixfalcon](https://docs.px4.io/en/flight_controller/pixfalcon.html)
-  * FMUv3 [Pixhawk 2](https://docs.px4.io/en/flight_controller/pixhawk-2.html)
-  * FMUv4
-    * [Pixracer](https://docs.px4.io/en/flight_controller/pixracer.html)
-    * [Pixhawk 3 Pro](https://docs.px4.io/en/flight_controller/pixhawk3_pro.html)
-  * FMUv5 (ARM Cortex M7)
-    * [Pixhawk 4](https://docs.px4.io/en/flight_controller/pixhawk4.html)
-    * [Pixhawk 4 mini](https://docs.px4.io/en/flight_controller/pixhawk4_mini.html)  
-  * [Airmind MindPX V2.8](http://www.mindpx.net/assets/accessories/UserGuide_MindPX.pdf)
-  * [Airmind MindRacer V1.2](http://mindpx.net/assets/accessories/mindracer_user_guide_v1.2.pdf)
-  * [Bitcraze Crazyflie 2.0](https://docs.px4.io/en/flight_controller/crazyflie2.html)
-  * [Omnibus F4 SD](https://docs.px4.io/en/flight_controller/omnibus_f4_sd.html)
-  * [BeagleBone Blue](https://docs.px4.io/en/flight_controller/beaglebone_blue.html)
-  * [Holybro Durandal](https://docs.px4.io/en/flight_controller/durandal.html)
-  * [Holybro Kakute F7](https://docs.px4.io/en/flight_controller/kakutef7.html)
-
-Additional information about supported hardware can be found in [PX4 user Guide > Autopilot Hardware](https://docs.px4.io/en/flight_controller/).
+
+* [Snapdragon Flight](https://docs.px4.io/en/flight_controller/snapdragon_flight.html)
```

Comment:
ALL the links should change from prefix `https://docs.px4.io/en/` to `https://docs.px4.io/master/en/`. That guarantees to point to latest version.

Reply:
Why is the other flight controller https://docs.px4.io/en/? Can you tell me how to do it?